vamp-server/test.sh

```diff
@@ -27,6 +27,7 @@ respfile="$tmpdir/resp.json"
 allrespfile="$tmpdir/resp.all"
 input="$tmpdir/input"
 expected="$tmpdir/expected"
+expected_less_strict="$tmpdir/obtained-less-strict"
```

Comment:
Ah, I had been erroneously performing the same thing to the obtained output at first before realising it needed to be done on the expected. I guess I missed this when changing the variable name. Will fix.